ecosystem.config.js

```diff
@@ -77,5 +77,23 @@ module.exports = {
         WRITE_AWS_S3_BUCKET_NAME : process.argv[7]
       }
     },
+    {
+      name      : "s3-landing-to-local",
+      script    : "/appdata/data-transfer/bin/data-transfer",
+      interpreter: "python",
+      env: {
+        INGEST_DEST_PATH : "/home/wherescape/test_for_data_transfer_ingest_landing",
```

Comment:
`test_for_*` why is it named lake that? Can it not be `data_transfer_ingest_landing` instead?

Reply:
this is the first pass to test the configuration, will be switched over to the appropriate directory name when the configuration is switched on in prod